core/runtime/src/main/java/io/quarkus/runtime/logging/ConsoleConfig.java

```diff
@@ -15,6 +15,12 @@
     @ConfigItem(defaultValue = "true")
     boolean enable;
 
+    /**
+     * If web stream console logging should be enabled. Enabled by default in dev mode for Dev UI.
+     */
+    @ConfigItem(defaultValue = "false")
+    boolean enableWebStream;
+
```

Comment:
This config exits so that logger-manager (in Quarkiverse) use this in Non Dev Mode.

Reply:
Then shouldn't this be a build item rather than a config option?

Reply:
Ok cool, how would that work ? An empty optional build item that if there, we should include in non-dev ?

Reply:
We usually use a  `MultiBuildItem` in this case in case multiple extensions try to enable the feature, In you build step you just check if the list of build items is empty or not to determine whether or not the feature has been turned on by some other extension